documentation/0005-node-difference-browser/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Differences between Node and the Browser
+description: 'How writing JavaScript application in Node.js differs from programming for the Web inside the browser'
+author: flaviocopes
+---
+
+Both the browser and Node use JavaScript as their programming language.
+
+Building apps that run in the browser is a completely different thing than building a Node.js application.
+
+Despite the fact that it's always JavaScript, there are some key differences that make the experience radically different.
+
+As a frontend developer that writes Node apps have a huge advantage - the language is still the same.
```

Comment:
This line feels a bit inconsistent.
May I suggest replacing it with something along the line of: `As a frontend developer who extensively uses Javascript, Node apps brings with it, a huge advantage - the comfort of programming everything, the frontend and the backend, in a single language`

Reply:
👍